src/WindowsInstaller/Product.wxs

```diff
@@ -125,17 +125,25 @@
               Checksum="yes">
           <Class Id="$(var.ProfilerCLSID)" Context="InprocServer32" ThreadingModel="both" Description="$(var.ProductNamePlatformAgnostic)"/>
         </File>
+        <File Id="LibSqreen"
+              Source="$(var.LibSqreenDirectory)\runtimes\win-x64\native\Sqreen.dll"
+              Checksum="yes">
+        </File>
       </Component>
     </ComponentGroup>
 
-    <!-- For the 64-bit installer, also install the 32-bit profiler -->
+    <!-- For the 64-bit installer, also install the 32-bit binaries -->
     <ComponentGroup Id="Files.Native.32" Directory="INSTALLFOLDER.32">
       <Component Win64="no" Id="Datadog.Trace.ClrProfiler.Native.32">
         <File Id="Datadog.Trace.ClrProfiler.Native.32"
               Source="$(var.TracerHomeDirectory)\win-x86\Datadog.Trace.ClrProfiler.Native.dll"
               Checksum="yes">
           <Class Id="$(var.ProfilerCLSID)" Context="InprocServer32" ThreadingModel="both" Description="$(var.ProductNamePlatformAgnostic)"/>
         </File>
+        <File Id="LibSqreen.32"
```

Comment:
How is Sqreen located by the Tracer? The only other file in this 32-bit directory is Datadog.Trace.ClrProfiler.Native.dll, which is usually found via the Profiler CLSID

Reply:
Yes, this is a bit tricky. There are several approaches, but after discussing it at Tuesday evening meeting we ended up deciding to use the DD env var to search for the dll and load it using system calls. It's implemented here: https://github.com/DataDog/dd-trace-dotnet/blob/appsec/v0-integration/src/Datadog.Trace/AppSec/Waf/NativeBindings/Native.cs#L38